### ISSUE TITLE
Cannot place "Blue Hurricane" (hyper coaster)

### ISSUE THREAD

**Issue by Pirenbaum** [OpenRCT2/OpenRCT2#12918]
Even on completely flattened land, when trying to place the prebuilt coaster "Blue Hurricane" it says "Cannot build that here".

**Comment by James103** [comment]
1. What version (including commit/build) are you attempting to place the coaster in?
2. Do you have the no height limit cheat set?
3. Do you have zero clearance turned on?
4. Can you upload logs?

**Comment by AaronVanGeffen** [comment]
Heh, not the first time we've had a bug report for this particular track design.

Could you fill out the issue template, though? This could be relevant (Version and build of OpenRCT2, and your operating system.)

**Comment by James103** [comment]
I just tested on 135cc10, and it turns out you need the no support limit cheat turned on to build the "Blue Hurricane" prebuilt ride. Is this intended?

**Comment by tombomp** [comment]
It's possible this is a result of #12707 - someone on the discord mentioned a few other rides which can no longer be built due to the collision box changes.

**Comment by Pirenbaum** [comment]
> 1. What version (including commit/build) are you attempting to place the coaster in?
> 2. Do you have the no height limit cheat set?
> 3. Do you have zero clearance turned on?
> 4. Can you upload logs?

Where can I find those?

I am on the latest update of OpenRCT2, no cheats.

**Comment by James103** [comment]
1. The version info can be found on the bottom left corner of the main menu.
2. Cheats can be viewed by the following: Open the options menu, go to the "Controls and interface" tab (with the twin gears icon), and tick the checkbox next to "Chests" to show the button to open the cheats menu. To enable the no support limit cheat, open the cheats dropdown and select "Disable support limits".
3. To enable the zero clearance cheat, follow the steps in (2) but select "Disable clearance checks" instead.
4. You may be able to pipe the log output to a file and then upload that file to act as your logs; the exact steps to do so depend on the OS.

**Comment by Fred-1044276** [comment]
Actually, Blue Hurricane can't be built because for some reason, it defaults to the Corkscrew Roller Coaster ride type instead of the Hypercoaster it should be. Same with Monster Trucks (becomes Car Ride), Hyper-Twister (becomes Twister Roller Coaster), Classic Mini Roller Coaster (becomes Junior Roller Coaster), and maybe the Spinning Wild Mouse (becomes Steel Wild Mouse?). This is because these aforementioned ride types were previously part of "ride groups" that have since been removed, with these ride types becoming their own.

**Comment by Pirenbaum** [comment]
> 1. The version info can be found on the bottom left corner of the main menu.
> 2. Cheats can be viewed by the following: Open the options menu, go to the "Controls and interface" tab (with the twin gears icon), and tick the checkbox next to "Chests" to show the button to open the cheats menu. To enable the no support limit cheat, open the cheats dropdown and select "Disable support limits".
> 3. To enable the zero clearance cheat, follow the steps in (2) but select "Disable clearance checks" instead.
> 4. You may be able to pipe the log output to a file and then upload that file to act as your logs; the exact steps to do so depend on the OS.

1. V.0.3.0-185-gcf5b48a9d(cf5b48a on develop)
2/3: no cheats enabled
4: windows 10

**Comment by Fred-1044276** [comment]
Here are screenshots proving that prebuilts involving the Monster Trucks, Hypercoaster, Hyper-Twister, Spinning Wild Mouse, and Classic Mini Roller Coaster are broken.
![Test Park 2020-09-12 19-04-32](https://user-images.githubusercontent.com/61885939/93008566-20bede80-f52b-11ea-8888-5aee8d601789.png)
![Test Park 2020-09-12 19-04-45](https://user-images.githubusercontent.com/61885939/93008568-24526580-f52b-11ea-8fe3-18f4e6a89310.png)
![Test Park 2020-09-12 19-05-11](https://user-images.githubusercontent.com/61885939/93008572-29171980-f52b-11ea-9de9-21067de9f340.png)
![Test Park 2020-09-12 19-05-27](https://user-images.githubusercontent.com/61885939/93008574-2ae0dd00-f52b-11ea-861c-3722ed5b89f2.png)
![Test Park 2020-09-12 19-06-35](https://user-images.githubusercontent.com/61885939/93008575-2caaa080-f52b-11ea-963b-b49d3501b19a.png)
Notice that they all magically revert back to their base ride types (Car Ride, Corkscrew Roller Coaster, Twister Roller Coaster, Steel Wild Mouse, and Junior Roller Coaster, respectively) instead of actually being their proper separate ride types (Monster Trucks, Hypercoaster, Hyper-Twister, Spinning Wild Mouse, and Classic Mini Roller Coaster, respectively). I used the "Allow arbitrary ride type changes" cheat to show the offending ride types that end up being used instead once these prebuilts have been placed down.

**Comment by Fred-1044276** [comment]
Here's a [zip file with the affected RCT2 base game prebuilts that I have fixed](https://drive.google.com/file/d/1VSjH_2viWibNorRbbha83udtpH9AfmFK/view?usp=sharing). I built all of them with the "Disable support limits" cheat on and changed them back to their correct ride types with the "Allow arbitrary ride type changes" cheat.
![Test Park 2020-09-13 10-44-46](https://user-images.githubusercontent.com/61885939/93024849-32959580-f5ae-11ea-95c9-a924c328e65e.png)
Here's Krypton Dips, now correctly a Hyper-Twister.
